src/members/ui/StatsBox.tsx

```diff
@@ -0,0 +1,101 @@
+import { Link } from 'react-router-dom';
+import styled from '@emotion/styled';
+
+import Text from '@/common-ui/Text';
+import { theme } from '@/styles/theme';
+import { numberWithCommas } from '@/utils/numberWithCommas';
+import Icon from '@/common-ui/assets/Icon';
+import getRem from '@/utils/getRem';
+
+const StatsBox = ({
+  numberOfLikes,
+  numberOfCategories,
+  numberOfComments,
+}: {
+  numberOfLikes: number;
+  numberOfCategories: number;
+  numberOfComments: number;
+}) => {
+  return (
+    <Container style={{ marginTop: getRem(2) }}>
+      <StatsContainer>
+        <Link to="/likes">
+          <StatItemContainer>
+            <IconLabelContainer>
+              <Icon name={'like'} size={'s'} />
+              <Text.Span style={{ marginLeft: '0.8rem' }}>좋아요</Text.Span>
```

Comment:
```suggestion
const IconText = styled(Text.Span)`
  margin-left: 0.8rem;
`

<IconText>좋아요</IconText>
```
[p2] 위와 같이 styled component로 선언해서 스타일링해줘도 좋을것같습니당